cmake/onnxruntime.cmake

```diff
@@ -68,7 +68,9 @@ if (NOT WIN32)
   endif()
 endif()
 
-if(${CMAKE_SYSTEM_NAME} MATCHES "Android")
+
+# strip binary on Android, or for a minimal build on Unix
+if(CMAKE_SYSTEM_NAME STREQUAL "Android" OR (onnxruntime_MINIMAL_BUILD AND UNIX))
```

Comment:
Should we also limit this to Release and MinSizeRel builds? Not sure that stripping a binary in a build that is meant to have debug info would be expected.